orchagent/intfsorch.cpp

```diff
@@ -1094,6 +1094,18 @@ void IntfsOrch::generateInterfaceMap()
     m_updateMapsTimer->start();
 }
 
+bool updateSyncdIntfPfx(const string& alias, const IpPrefix &ip_prefix)
```

Comment:
class name - `IntfsOrch  `missing in definition

Reply:
Yes, mistakanly uploaded wrong patch. Fixed it.